workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedGetAppPage/PurchasedGetAppPage.tsx

```diff
@@ -0,0 +1,600 @@
+import ClayForm, {ClayCheckbox, ClayInput} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClayLink from '@clayui/link';
+import {InputHTMLAttributes, useEffect, useMemo, useState} from 'react';
+import {useForm} from 'react-hook-form';
+import {z} from 'zod';
+
+import {Header} from '../../components/Header/Header';
+import BaseWrapper from '../../components/Input/base/BaseWrapper';
+import zodSchema, {zodResolver} from '../../schema/zod';
+import {
+	getAccountInfo,
+	getListTypeDefinitionByExternalReferenceCode,
+	getProductById,
+	getProductSKU,
+	getUserAccount,
+	postAccountByERCUserAccountByERC,
+} from '../../utils/api';
+
+import './PurchasedGetAppPage.scss';
+
+import ClaySticker from '@clayui/sticker';
+
+import emptyPictureIcon from '../../assets/icons/avatar.svg';
+import {Footer} from '../../components/Footer/Footer';
+import Select from '../../components/Select/Select';
+import fetcher from '../../services/fetcher';
+import CreatedProjectCard from './CreatedProjectCard';
+import PurchasedGetAppAccountSelection from './PurchasedGetAppAccountSelection';
+import {getPhones} from './PurchasedGetAppPageUtil';
+
+type Steps = {
+	page:
+		| 'accountCreation'
+		| 'accountSelection'
+		| 'projectCreated'
+		| 'initialStep';
+};
+
+const accountTypes = {
+	BUSINESS: 'business',
+	PERSON: 'person',
+};
+
+export type UserForm = z.infer<typeof zodSchema.accountCreator>;
+
+type InputProps = {
+	boldLabel?: boolean;
+	className?: string;
+	description?: string;
+	disabled?: boolean;
+	errors?: any;
+	id?: string;
+	label?: string;
+	name: string;
+	options?: {label: string; value: string} | [];
+	register?: any;
+	required?: boolean;
+	type?: string;
+} & InputHTMLAttributes<HTMLInputElement>;
+
+const {origin} = window.location;
+const externalReferenceCode = 'MKT-INDUSTRIES';
+
+const Input: React.FC<InputProps> = ({
+	boldLabel,
+	className,
+	description,
+	disabled = false,
+	errors = {},
+	label,
+	name,
+	register = () => {},
+	id = name,
+	type,
+	value,
+	required = false,
+	onBlur,
+	options,
+	...otherProps
+}) => {
+	return (
+		<BaseWrapper
+			boldLabel={boldLabel}
+			description={description}
+			disabled={disabled}
+			error={errors[name]?.message}
+			id={id}
+			label={label}
+			required={required}
+		>
+			<ClayInput
+				className="rounded-xs"
+				component={type === 'textarea' ? 'textarea' : 'input'}
+				disabled={disabled}
+				id={id}
+				name={name}
+				type={type}
+				value={value}
+				{...otherProps}
+				{...register(name, {onBlur, required})}
+			/>
+		</BaseWrapper>
+	);
+};
+
+const PurchasedGetAppPage: React.FC = () => {
+	const queryString = window.location.search;
+
+	const urlParams = new URLSearchParams(queryString);
+
+	const productId = Number(urlParams.get('productId')) + 1;
+
+	const [step, setStep] = useState<Steps>({page: 'initialStep'});
+	const [phonesFlags, setPhonesFlags] = useState<PhonesFlags[]>();
+	const [currentphonesFlags, setCurrentPhonesFlags] = useState({
```

Comment:
user CamelCase to maintain the pattern
```suggestion
	const [currentPhonesFlags, setCurrentPhonesFlags] = useState({
```